stored_requests/fetcher.go

```diff
@@ -6,11 +6,16 @@ import (
 	"fmt"
 )
 
-// Fetcher knows how to fetch Stored Request data by id.
+type GlobalFetcher interface {
+	FetchRequests(ctx context.Context, requestIDs []string, impIDs []string) (requestData map[string]json.RawMessage, impData map[string]json.RawMessage, errs []error)
+	FetchCategories(primaryAdServer, publisherId, iabCategory string) (string, error)
+}
+
+// StoredRequestsFetcher knows how to fetch Stored Request data by id.
 //
 // Implementations must be safe for concurrent access by multiple goroutines.
 // Callers are expected to share a single instance as much as possible.
-type Fetcher interface {
+type StoredRequestsFetcher interface {
```

Comment:
Why the rename to `StoredRequestsFetcher`? The reference outside of the package is `stored_request.StoredRequestFetcher`, which make the addition of `StoredRequest` redundant.